debezium-storage/debezium-storage-jdbc/src/test/java/io/debezium/storage/jdbc/history/JdbcSchemaHistoryIT.java

```diff
@@ -147,7 +148,7 @@ private JdbcConnection testConnection() {
                 .withDatabase(DBNAME)
                 .build();
         final String url = "jdbc:mysql://${hostname}:${port}/${dbname}";
-        return new JdbcConnection(jdbcConfig, JdbcConnection.patternBasedFactory(url), "`", "`");
+        return new JdbcConnection(jdbcConfig, JdbcConnection.patternBasedFactory(url), "`", "`", new ThreadNameContext("test-connector","${debezium}-${connector.name}-${task.id}","0"));
```

Comment:
Let's add ${connector.name} and ${task.id} on top of the default thread name pattern.